ddterm/org.gnome.Shell.Extensions.ddterm.service

```diff
@@ -0,0 +1,3 @@
+[D-BUS Service]
+Name=org.gnome.Shell.Extensions.ddterm
```

Comment:
There's currently nothing with well-known name `org.gnome.Shell.Extensions.ddterm`. The app registers on the bus under the name `com.github.amezin.ddterm` https://github.com/ddterm/gnome-shell-extension-ddterm/blob/master/ddterm/app/application.js#L310

Reply:
In e0c2db5, I changed the well-known name (and the filename to match it).